app/views/comments/_form.html.erb

```diff
@@ -83,7 +83,17 @@
           <script>
             jQuery(document).ready(function() {
               $E.initialize();
+              $('button[id="btn-publish"]').attr('disabled', true);
+              $('textarea[class="form-control"]').on('keyup', function () {
+                var comment = $(this).val();
+                if (comment != '') {
```

Comment:
@ValeriaCobianISW this is pone to errors. For example, I type few spaces, or press tab key, this will enable the submit button, while it shouldn't because there is nothing in the textarea, except blank space, of course.

Reply:
Yeah, you should use something like `if (comment.trim.length == 0)`.
The trim removes the whitespace